pkg/apiserver/registry/inventory/virtualmachine/rest.go

```diff
@@ -64,6 +64,9 @@ func (r *REST) New() runtime.Object {
 	return &runtimev1alpha1.VirtualMachine{}
 }
 
+func (r *REST) Destroy() {
```

Comment:
Is this a new requirement for upgrading?

Reply:
Yes.. with antrea 1.12, runtime version is upgraded too.